Modules/Microsoft.PowerShell.UnixCompleters/Microsoft.PowerShell.UnixCompleters.psd1

```diff
@@ -0,0 +1,127 @@
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = 'Microsoft.PowerShell.UnixCompleters.dll'
```

Comment:
Just a convention of the modules here. I thought I'd follow it, but happy to change it if there's a particular reason why we should

Reply:
I believe it should be in Contribution docs.

Reply:
When we publish to PSGallery, we require the `Microsoft` namespace.  @iSazonov, can you open a new issue to indicate this requirement in the Contribution docs for new modules?  We are sticking with the `Microsoft.PowerShell.*` format for naming as other Microsoft teams may use `Microsoft.*` to avoid conflicts.

Reply:
Done #52